python/py-pyside2/Portfile

```diff
@@ -4,8 +4,8 @@ PortSystem              1.0
 PortGroup               python 1.0
 
 name                    py-pyside2
-version                 5.15.2
-revision                1
+version                 5.15.2.1
```

Comment:
latest upstream release is 5.15.3 - can you please update to that version?

Reply:
done